libtrip-notification/api/current.txt

```diff
@@ -0,0 +1,32 @@
+// Signature format: 3.0
+package com.mapbox.navigation.trip.notification {
+
+  @com.mapbox.annotation.module.MapboxModule(type=MapboxModuleType.NavigationTripNotification) public final class MapboxTripNotification implements com.mapbox.navigation.base.trip.notification.TripNotification {
+    ctor public MapboxTripNotification(android.content.Context applicationContext, com.mapbox.navigation.base.options.NavigationOptions navigationOptions);
+    method public String? getCurrentManeuverModifier();
+    method public String? getCurrentManeuverType();
+    method public android.app.Notification getNotification();
+    method public int getNotificationId();
+    method public void onTripSessionStarted();
+    method public void onTripSessionStopped();
+    method public void updateNotification(com.mapbox.navigation.base.trip.model.RouteProgress routeProgress);
+    property public final String? currentManeuverModifier;
+    property public final String? currentManeuverType;
+    field public static final com.mapbox.navigation.trip.notification.MapboxTripNotification.Companion! Companion;
+  }
+
+  public static final class MapboxTripNotification.Companion {
+    method public kotlinx.coroutines.channels.Channel<com.mapbox.navigation.base.trip.notification.NotificationAction> getNotificationActionButtonChannel();
+    method public void setNotificationActionButtonChannel(kotlinx.coroutines.channels.Channel<com.mapbox.navigation.base.trip.notification.NotificationAction> p);
+    property public final kotlinx.coroutines.channels.Channel<com.mapbox.navigation.base.trip.notification.NotificationAction> notificationActionButtonChannel;
+  }
+
+}
+
+package com.mapbox.navigation.trip.notification.time.span {
+
+  public final class SpanExKt {
```

Comment:
Could we avoid tracking this empty `public` class? 🤔 There are only top-level functions

Reply:
IMO we should track it to be more clean, but also we might use `@file:JvmName("ClassName")` to be more explicitly  
it's related for all similar notes above